core/src/AbilityHelpers.ts

```diff
@@ -264,3 +282,12 @@ export function getAbilityHelpersAttribute(
         [Types.AbilityHelpersAttributeName]: attr
     };
 }
+
+
+export function getCurrentAbilityHelpers(win: WindowWithAHInstance): Types.AbilityHelpersCore | null {
+    if (win && win.__ahInstance) {
```

Comment:
You use optional chaining above, why not use it here like `return win?.__ahInstance || null` for more compactness.

Reply:
it's a reflex  when writing if conditions, will remove